author_handlers_test.go

```diff
@@ -65,6 +69,25 @@ func TestShouldReturn200AndAuthorsCount(t *testing.T) {
 	assert.Equal(t, "2", actualOutput, "Response body should contain the count of available authors")
 }
 
+func TestShouldReturn500WhenAuthorsCountIsCalledAndCacheRefreshFails(t *testing.T) {
+	mbs := new(MockedBerthaService)
+	mbs.On("getAuthorsCount").Return(2)
+	mbs.On("refreshCache").Return(errors.New("I hate Luca!"))
+	startCuratedAuthorsTransformer(mbs)
+	defer curatedAuthorsTransformer.Close()
+
+	resp, err := http.Get(curatedAuthorsTransformer.URL + "/transformers/authors/__count")
+	if err != nil {
```

Comment:
Would you really panic() in a test case? Maybe just asserting that err is nil would be sufficient.